pool/app/settings/settings.ml

```diff
@@ -128,3 +128,5 @@ let terms_and_conditions pool language =
   |> CCOption.to_result Pool_common.Message.(Retrieve Field.TermsAndConditions)
   |> Lwt_result.lift
 ;;
+
+let find_sender _ = failwith "todo"
```

Comment:
may use currently smtp username ("Pool_tenant.SmtpAuth.Username") from the tenant (@aronerben ?)

(no empty new line)